nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/bin/nifi.sh

```diff
@@ -118,6 +118,12 @@ locateJava() {
             fi
         fi
     fi
+    [ "x${TOOLS_JAR}" =  "x" ] && [ -n "${JAVA_HOME}" ] && TOOLS_JAR=$(find "${JAVA_HOME}" -name "tools.jar")
+    [ "x${TOOLS_JAR}" =  "x" ] && TOOLS_JAR=$(find "${JAVA_HOME}" -name "classes.jar")
+    if ["x${TOOLS_JAR}" =  "x" ]; then
```

Comment:
I think my preference would be to opt to only display this if we are using the env command.